great_expectations/expectations/expectation.py

```diff
@@ -1166,11 +1151,22 @@ def get_validation_dependencies(
             metric_configurations={}, result_format=result_format
         )
 
+    def _get_default_value(self, key: str) -> Any:
+        """Get default value for the model if it is not required.
+        NOTE: For reasons I'm unclear on, we can get back either a FieldInfo or a ModelField.
```

Comment:
You're still seeing this? I would this `self.__fields__` would give us back a defined structure

Reply:
oops! this was old. Removing shortly.